README.md

```diff
@@ -114,4 +109,10 @@ $ docker run --env-file ./.env -d --publish 5000:5000 --mount type=bind,source="
 $ docker build --target production --tag do-me:prod .
 $ docker run --env-file ./.env -d --publish 5000:5000 do-me:prod
 ```
+
+To launch in interactive mode add the -it flag:
+```bash
+$ docker run --env-file ./.env -it --publish 5000:5000 --mount type=bind,source="$(pwd)"/todo_app,target=/do-me/todo_app do-me:dev
+$ docker run --env-file ./.env -it --publish 5000:5000 do-me:prod
```

Comment:
There's nothing wrong about this, but I'm not sure how helpful it is (other than letting you more easily kill the container anyway!)
This might have been added with respect to your previous entrypoint which launched bash, in which case overriding the default entrypoint so that this is for using bash could be what you want now:
```bash
docker run --env-file ./.env -it --publish 5000:5000 --entrypoint bash do-me:prod
```